### ISSUE TITLE
Python import list depend will got a error when run script

### ISSUE THREAD

**Issue by HATTER-LONG** [michaelb/sniprun#283]
import like this then run python script with sniprun will get error

```python
from PySide6.QtWidgets import (
    QApplication,
    QMainWindow,
    QStackedLayout,
    QWidget,
)
```
![image](https://github.com/michaelb/sniprun/assets/31438141/646e9711-330f-422d-b204-bfa7eb18eaab)

the run code generated by sniprun of  ~/Library/Caches/sniprun/python3_original/main.py:

```python
try:
	
	import sys
	from PySide6.QtGui import QColor, QPalette
	from PySide6.QtWidgets import (

except:
	pass

```

- Neovim 0.9.5
- macos

**Comment by michaelb** [comment]
Indeed, I probably missed this way of doing python imports

Will fix (someday, hopefully not too far in the future)

**Comment by michaelb** [comment]
This issue should be fixed by the latest (v1.3.13) release.
Don't hesitate to reopen this issue if you still experience problems related to the described bug
